lib/utils/parseNumber.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export function parseNumber(value) {
```

Comment:
I think you need to put the purpose of this function, cause it's not obvious from the name. Please pay attention that it's a lib with common components. 
as for me `parseNumber` is a `Number(value)`. So it's better to describe (in a name or comment) why it's needed

Reply:
yep, not clear, do we really need such util?

Reply:
I think we don't need this util now, because behavior has not changed with or without it.

Reply:
we definitely need this code due to specific validation rules of the textfield. However, naming is quite confusing

Reply:
renamed to `parseNumberFieldValue`